### PR TITLE
MOST-166: Added EVM_MAINNET key to hardhat config. 

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
Supplied Testnet configuration used in the past as well.

### CODE REVIEW

File: azero/env/mainnet.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  "ws_node": "wss://ws.azero.dev",
+  "relayers": [
+    "<AZERO deployer address>"
+  ],
+  "deployer_seed": "<AZERO deployer seed>",
+  "signature_threshold": 1,
+  "dev": false,
+  "azero_deposit": 100000,
+  "token_config_path": "/../../cfg/tokens_mainnet.json"
+}
```
Comment (kostekIV) [inline azero/env/mainnet.json:11]: nit add a newline

File: azero/env/testnet.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "ws_node": "wss://ws.test.azero.dev",
+  "relayers": [
+    "<TZERO deployer address>"
+  ],
+  "deployer_seed": "<TZERO deployer seed>",
+  "signature_threshold": 1,
+  "dev": false,
+  "azero_deposit": 100000,
+  "token_config_path": "/../../cfg/tokens_testnet.json"
+}
```
Comment (kostekIV) [inline azero/env/testnet.json:11]: here also

File: cfg/tokens_mainnet.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "aleph": [
+    {
+      "name": "Wrapped AZERO",
+      "symbol": "wAZERO",
+      "decimals": 12,
+      "deployed": true,
+      "address": "5CtuFVgEUz13SFPVY6s2cZrnLDEkxQXc19aXrNARwEBeCXgg"
+    }
+  ]
+}
```
Comment (kostekIV) [inline cfg/tokens_mainnet.json:12]: add them everywhere :P

**Review by kostekIV — CHANGES_REQUESTED** [review]
Please rename configs/envs to have `l2` prefix

File: azero/env/mainnet.json (hunk 1 above)
Comment (Marcin-Radecki) [reply azero/env/mainnet.json:11]: done

File: azero/env/testnet.json (hunk 2 above)
Comment (Marcin-Radecki) [reply azero/env/testnet.json:11]: done

**Review by kostekIV — APPROVED** [review]

**Pull request merged**